views/mutators.qtpl

```diff
@@ -7,16 +7,20 @@
 <main class="main-width edit edit_no-preview">
 	<h1 class="edit__title">Edit {%s util.BeautifulName(hyphaName) %}</h1>
 	{%s= warning %}
+	<div class="toolbar">
+		<button onclick="insertDate()">Insert Date</button>
+	</div>
 	<form method="post" class="edit-form"
 			action="/upload-text/{%s hyphaName %}">
-		<textarea name="text">{%s textAreaFill %}</textarea>
+		<textarea id="edit-textarea" name="text">{%s textAreaFill %}</textarea>
```

Comment:
The id should be something like `edit__textarea`, we're trying to follow BEM :-)